app/components/cron-job.js

```diff
@@ -8,6 +8,46 @@ export default Ember.Component.extend({
   actionType: 'Save',
   store: service(),
 
+  intervalText: function() {
```

Comment:
@renee-travisci Thanks for such clear instructions. I had mentioned breaking this up before, but this is much more actionable.